### PR TITLE
Bump react-palm

### PR DESCRIPTION
**Pull request by benshope**
Bumps react-palm to the latest version

### CODE REVIEW

**Comment by btford** [comment]
Your yarn.lock file is incorrect (for some reason), which is part of why CI is failing.

**Comment by benshope** [comment]
Cool, changed the yarn.lock and that made it pass

**Comment by btford** [comment]
Can you squash the commits into one? @heshan0131 can you give this a look?

Thanks!

**Comment by ericsoco** [comment]
Oh also should there be a version bump / patch? Not sure how strictly Kepler follows semver.

**Comment by heshan0131** [comment]
Please also update the code example in README.md
change your`./npmrc` to `registry = https://registry.yarnpkg.com/` then run `yarn` to update `yarn.lock`

**Comment by benshope** [comment]
@ericsoco @heshan0131 at least some of the `yarn.lock` files are not updated because of `.gitignore`.  Should I remove those lines in the `.gitignore`?

**Comment by heshan0131** [comment]
we only include 1 yarn.lock at the root, that's the only one need to be updated

File: yarn.lock [hunk 1]
```diff
@@ -5925,11 +5925,9 @@ react-motion@^0.5.2:
...
   resolved "https://registry.npmjs.org/react-onclickoutside/-/react-onclickoutside-6.7.1.tgz#6a5b5b8b4eae6b776259712c89c8a2b36b17be93"
 
-react-palm@^0.8.1:
-  version "0.8.1"
-  resolved "https://registry.yarnpkg.com/react-palm/-/react-palm-0.8.1.tgz#7c7fd7348235ea04fd46c5da4a11000e0ee9343d"
-  dependencies:
-    prop-types "^15.6.0"
+react-palm@^1.1.2:
+  version "1.1.2"
+  resolved "https://unpm.uberinternal.com/react-palm/-/react-palm-1.1.2.tgz#3c6262fbbb37c9e6f844d65d5086dc67ec5fae8e"
```
Comment (heshan0131) [inline yarn.lock]: it is still using `unpm`, did you change your `~/.npmrc` ?

Comment (benshope) [reply yarn.lock]: oh that's strange - I checked my registry with `yarn get registry` and it looked right.  I'll do it by changing the file

**Pull request merged**